### ISSUE TITLE
java.lang.ClassCastException occurred.

### ISSUE THREAD

**Issue by freest4r** [SwitchEV/RISE-V2G#88]
[RISE_SECC_log.txt](https://github.com/SwitchEV/RISE-V2G/files/8490130/RISE_SECC_log.txt)
[ChargeParameterDiscoveryReq.txt](https://github.com/SwitchEV/RISE-V2G/files/8490141/ChargeParameterDiscoveryReq.txt)

Recently I'm working on testing ISO 15118-2.
Thanks to RISE, I can do my research well. thank you.

To test _rise-v2g-secc-1.2.6.jar_, I implement EVCC that transmits unexpected XMLs.
In this process, I found several errors and today I would like to report the one of the errors.

In "**ChargeParameterDiscoveryReq**" step, 
if "**DepartureTime**" of "**AC_EVChargeParameter**" is empty (**<ns3:DepartureTime></ns3:DepartureTime>**), 
**RuntimeException** is occurred in _rise-v2g-secc-1.2.6.jar_.
but it's OK because  _rise-v2g-secc-1.2.6.jar_ send "FAILED_SequenceError" to EVCC.

At this time, 
If EVCC connect to _rise-v2g-secc-1.2.6.jar_ and send normal **SupportedAppProtocolReq**, 
then "**java.lang.ClassCastException**" is occurred.

The exception message is like below:

_java.lang.ClassCastException: class com.v2gclarity.risev2g.shared.v2gMessages.appProtocol.AppProtocolType cannot be cast to class javax.xml.bind.JAXBElement (com.v2gclarity.risev2g.shared.v2gMessages.appProtocol.AppProtocolType and javax.xml.bind.JAXBElement are in unnamed module of loader 'app')_

Then, _rise-v2g-secc-1.2.6.jar_ send "**Failed_NoNegotiation**" to the EVCC.
Even though the **SupportedAppProtocolReq** message was normal.

I'm not sure why it happens, 
but I attached logs and ChargeParameterDiscoveryReq.xml files.

In summary, the reproduce process is as follows:

1. run _rise-v2g-secc-1.2.6.jar_
2. EVCC connect to the _rise-v2g-secc-1.2.6.jar_ and continues the process normally until the **ChargeParameterDiscoveryReq** step.
3. At **ChargeParameterDiscoveryReq** step, send attached ChargeParameterDiscoveryReq.xml file to _rise-v2g-secc-1.2.6.jar_.
4.  **java.lang.RuntimeException** occurred -> **FAILED_SequenceError** -> Connection closed
5. EVCC connect to the _rise-v2g-secc-1.2.6.jar_ again and send normal **SupportedAppProtocolReq** message.
6. **java.lang.ClassCastException** occurred -> Failed_NoNegotiation -> connection closed
